learn/security/master_api_keys.md

```diff
@@ -8,6 +8,10 @@ This guide will teach you how to protect your Meilisearch instance by setting a
 
 To protect a Meilisearch instance from unauthorized use, you must supply a master key at launch. This master key can be of any length, but must be composed of valid UTF-8 characters.
 
+::: warning
+You need to set a master key to access the [`/keys`](/reference/api/keys.md) route. Otherwise you will get a [`missing_master_key`](/reference/errors/error_codes.md#missing-master-key) error.
```

Comment:
Do we need a comma here? Not sure 🤔 

```suggestion
You need to set a master key to access the [`/keys`](/reference/api/keys.md) route. Otherwise, you will get a [`missing_master_key`](/reference/errors/error_codes.md#missing-master-key) error.
```

Reply:
We do